integration_tests/auction_test.go

```diff
@@ -106,7 +107,10 @@ func (s *IntegrationTestSuite) TestAuctionModule() {
 		s.T().Log("Waiting for proposal to be approved..")
 		s.Require().Eventually(func() bool {
```

Comment:
I believe the format here with a string at the end is supposed to be `Eventuallyf`.

Reply:
I think the `f` is only for string interpolation in the error message? this one is just a string literal

Reply:
Yeah, I guess this does work for static strings, although the final argument being an empty interface called `msgAndArgs` is a little confusing.